api/external/exports.html

```diff
@@ -262,6 +262,10 @@ <h1>Exports<a class="headerlink" href="#exports" title="Permalink to this headli
         <br />
         <li><strong>member_status_ids</strong> (<em>array of strings</em>) &#8211; Status of the members</li>
         <li><strong>include_fields</strong> &#8211; Accepts True or an array of strings. Optional flag to include all member fields or a list of member fields.</li>
+        <li><strong>sms</strong> (<em>dictionary</em>) &#8211; Describes what should be exported from sms related fields. </li>
```

Comment:
I'm thinkin' go ahead and uppercase this bit for consistency:


```suggestion
        <li><strong>sms</strong> (<em>dictionary</em>) &#8211; Describes what should be exported from SMS related fields. </li>
```

... the actual attribute makes sense lowercase, but I think capitalizing everywhere that's not code is probably good. 😅

Reply:
Fixed it!